app/views/system_admin/dead_jobs/index.html.erb

```diff
@@ -30,8 +30,9 @@
               <%= render GovukButtonLinkTo::View.new(
                 body: "View",
                 url: dead_job_path(service.class),
+                id: "view_#{service.name.gsub(" ", "_").downcase}",
```

Comment:
each service has a method that looks like it does this just call `service.identifier`

Reply:
although you'll need to move the method out of private (in `dead_jobs/base.rb`)